History.txt

```diff
@@ -1,3 +1,6 @@
+== 2.0.0
```

Comment:
If all these changes are backwards-compatible, can we bump the minor version instead of the major?